apis/python/tests/_util.py

```diff
@@ -0,0 +1,17 @@
+from contextlib import contextmanager
+from typing import Type
+
+import pytest
+from typeguard import suppress_type_checks
+
+
+@contextmanager
+def raises(exc: Type[Exception]):
```

Comment:
This might be better named `raises_no_typeguard` or something like that, because otherwise it’s not immediately obvious what the difference between `with raises(...)` vs. `with pytest.raises(...)` is.

I would also recommend just forwarding all of `*args` and `**kwargs` to `pytest.raises`, so that the two are otherwise completely compatible.